test/Fondue.test.js

```diff
@@ -290,3 +290,21 @@ test("Extracts charset", async () => {
 		},
 	]);
 });
+
+describe("Layout features", () => {
```

Comment:
Do these two test actually cover all added code?

Reply:
It tests for either one of two outcomes: whether _some_ layout feature is returned for a font that has it, and that nothing is return for a font without layout features.

I could add tests to see specifically which layout feature is returned (e.g. single substitutions _and_ ligatures) but I would feel this belongs on the Font.js side.